www/includes/easyparliament/templates/html/static/privacy.php

```diff
@@ -39,50 +163,13 @@
 
             <p>Google Analytics stores information such as what pages you visit,
             how long you are on the site, how you got here, what you click on,
-            and information about your web browser.</p>
```

Comment:
survey questions question not removed a bit above here. And GA text doesn't match.